app/extensions/safe/auth-web-app/components/app_list.js

```diff
@@ -104,10 +98,10 @@ export default class AppList extends Component {
 
   getNoAppsContainer() {
     return (
-      <div className="no-apps">
+      <div key={"noApps"} className="no-apps">
         <h3 className="no-apps-h">Looks like you haven&rsquo;t authorised any apps yet.</h3>
-        <div className="no-apps-img">{''}</div>
-        <div className="no-apps-down">
+        <div key={"noAppsIMG"} className="no-apps-img">{''}</div>
```

Comment:
Noted. So, I ended up adding the keys for all the tags because prior to your commit for push Route warning, this was the only way to suppress the warning and finding the next place where the warning was present. I have  refactored it and will commit it after a round of testing.